charts/operator/testdata/template-cases.golden.txtar

```diff
@@ -226884,7 +226884,7 @@ spec:
         - YJt9t
         ip: UYcKtV
       - ip: 4uIxYAS
-      - {}
+      - ip: ""
```

Comment:
This feels a bit suspect. Any idea where this change is coming from?

Reply:
I have no idea. I believe some of the changes in transpiler due to the fact we have multiple go modules.